app/views/tasks/_in_progress_tasks.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="in_progress_tasks">
+  <table class="table table-bordered">
+    <thead>
+    <tr>
+      <td></td>
+      <td>In progress</td>
+      <td></td>
+    </tr>
+    </thead>
+    <tbody>
+    <% @tasks_started.each do |task| %>
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/2/files#r27945248)